node_mongo_tutorial/resources/index.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");
```

Comment:
### what is the use of this file?

- try to follow the suggested file/folder structure for better presentation.